Packs/UpdateObject/Scripts/UpdateObject/UpdateObject.py

```diff
@@ -0,0 +1,28 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: F401
+
+original_object = demisto.args()['original_object']
+key_to_update = demisto.args()['key_to_update']
+values = demisto.args()['values']
+index = int(demisto.args().get('index', '-1'))
+context_key = demisto.args().get('context_key', 'UpdatedObject')
+
+if not isinstance(values, list):
+    values = values.split(',')
+
+if index != -1:
+    original_object[index][key_to_update] = values[0]
```

Comment:
if original_object is a dict you'll get a KeyError, please fix it.

Reply:
Addressed in https://github.com/xsoar-contrib/content/pull/2

Reply:
Now, original_dict is a list (when you implement the above suggestion) but one of the items in the list isn't a dict you'll get an error again, please fix it